codegen-server/python/src/main/kotlin/software/amazon/smithy/rust/codegen/server/python/smithy/protocols/PythonServerProtocolLoader.kt

```diff
@@ -61,6 +61,29 @@ class PythonServerAfterDeserializedMemberServerHttpBoundCustomization() :
         is ServerHttpBoundProtocolSection.AfterTimestampDeserializedMember -> writable {
             rust(".into()")
         }
+
+        else -> emptySection
+    }
+}
+
+/**
+ * Customization class used to determine the type of serialized stream payload and how it should be wrapped in a
+ * new-type wrapper to enable `futures_core::stream::Stream` trait.
+ */
+class PythonServerStreamPayloadSerializerCustomization() : ServerHttpBoundProtocolCustomization() {
+    override fun section(section: ServerHttpBoundProtocolSection): Writable = when (section) {
+        is ServerHttpBoundProtocolSection.TypeOfSerializedStreamPayload -> writable {
+            // `aws_smithy_http_server_python::types::ByteStream` already implements
+            // `futures::stream::Stream`, so no need to wrap it in a futures' stream-compatible
+            // wrapper.
+            rust("#T", section.params.symbolProvider.toSymbol(section.params.member))
+        }
+
+        is ServerHttpBoundProtocolSection.WrapStreamPayload -> writable {
+            // payloadName is always non-null within WrapStreamAfterPayloadGenerated
```

Comment:
If it's always non-null, why is `payloadName` nullable in `StreamPayloadSerializerParams`?

Reply:
I refactored in e076f64 so that `payloadName` is not used by `StreamPayloadSerializerParams`. The refactoring is about wrapping stream payload with a `Stream`-trait-enabling new type now takes place at the call sites of a serializer rather than within the serializer itself.

That effectively removed the need for the `StreamPayloadSerializerRenderer` interface, since rendering customization can be hard-coded on the server side (and no customization is needed on the client side).